src/widgets/ErrorDialog.h

```diff
@@ -46,11 +46,15 @@ class HtmlTextHelpDialog : public BrowserFrame
 public:
    HtmlTextHelpDialog() : BrowserFrame()
    {
+#if !wxCHECK_VERSION(3, 1, 0)
```

Comment:
What happens if we just bypass MakeModal this way?  Does it leave the dialog modeless?  Will that allow unwanted interaction with the project window?